### PR TITLE
Feature: base structure update

### PR DESCRIPTION
**Pull request by ninibear030**

### CODE REVIEW

File: apps/hub/src/app/governance/governance-genre-helper.tsx [hunk 1]
```diff
@@ -1,47 +1,60 @@
...
 
-export type Dapp = {
+export type DappConfig = {
+  id: PROPOSAL_GENRE;
   color: string;
   icon: React.ReactNode;
   name: string;
   link: string;
+  subgraph: string;
 };
```
Comment (bearpong) [inline apps/hub/src/app/governance/governance-genre-helper.tsx:13]: what about renaming `DappConfig` to `GovernanceTopic`? I think it'd be even more intuitive and scoped. Also I was thinking that we might rename `link` to `slug`

I like a lot that we added an `id`, can you use it in useCreateProposal when setting the topic? So we're free to change the slug in the future

File: apps/hub/src/app/governance/[genre]/page.tsx [hunk 2]
```diff
@@ -1,27 +1,6 @@
...
-} from "../governance-genre-helper";
 import GovernanceByStatus from "./components/governance-by-status";
-import { governorAddress } from "@bera/config";
-
-export default function Page({
-  params,
-}: {
-  params: { genre: PROPOSAL_GENRE };
-}) {
-  if (!isValidGenre(params.genre)) return notFound();
```
Comment (bearpong) [inline apps/hub/src/app/governance/[genre]/page.tsx:19]: why did u remove this check?

**Review by bearpong — APPROVED** [review]
LGTM, I added some comments which might enhance it a lil more

File: apps/hub/src/app/governance/[genre]/page.tsx (hunk 2 above)
Comment (ninibear030) [reply apps/hub/src/app/governance/[genre]/page.tsx:19]: i put it in the provider ^

File: apps/hub/src/app/governance/governance-genre-helper.tsx (hunk 1 above)
Comment (ninibear030) [reply apps/hub/src/app/governance/governance-genre-helper.tsx:13]: yes ser

**Pull request merged**